### DISCUSSION TITLE
Welcome to discussions!

### DISCUSSION THREAD

**Discussion by jmduke** [buttondown/discussions#1] [Announcements]
## 👋 Welcome!
  I'm using Discussions as a place to connect and broadcast to Buttondown's users. I hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by tarob0ba** [comment]
I’m Beckett. I do stuff and things.

**Comment by JayCuthrell** [comment]
I'm Jay. I hope to contribute at some point.
